webpack.config.ts

```diff
@@ -22,25 +22,6 @@ module.exports = (_env: string, argv: Record<string, boolean | number | string>)
 	},
 	module: {
 		rules: [
-			{
-				test: /options-storage/,
-				loader: 'string-replace-loader',
-				options: {
-					search: '__featuresList__',
-					replace: () => {
-						const features = [];
-
-						const directoryPath = path.join(__dirname, 'source/features');
-						for (const filename of readdirSync(directoryPath)) {
-							if (filename.endsWith('.tsx')) {
-								features.push(filename.replace('.tsx', ''));
-							}
-						}
-
-						return JSON.stringify(features);
-					}
-				}
-			},
```

Comment:
I'm also moving this to `webpack.DefinePlugin` but I'm leaving `string-replace-loader` in the dependencies to enable #2119